src/Explorer/Tree/ResourceTreeAdapter.tsx

```diff
@@ -273,11 +274,7 @@ export class ResourceTreeAdapter implements ReactAdapter {
       contextMenu: ResourceTreeContextMenuButtonFactory.createCollectionContextMenuButton(this.container, collection),
     });
 
-    if (
-      userContext.apiType === "Mongo" &&
-      this.container.isNotebookEnabled() &&
-      userContext.features.enableSchemaAnalyzer
-    ) {
+    if (SchemaAnalyzerUtils.isSchemaAnalyzerSupported(this.container.isNotebookEnabled())) {
```

Comment:
>isSchemaAnalyzerSupported [](http://example.com/codeflow?start=28&length=25)

should this really be within schemaUtils?

gn that you pass notebook enabled as a apram alone and get usercontext and other directly.. since this file itself has usercontext and container, keep it as a fn here.. 
if u think u ll use it in multiple places - then makes sense to move it to SchemaAnalyzerUtils?